### ISSUE TITLE
The monitoring plugin removes carriage returns (\n) in text body 

### ISSUE THREAD

**Issue by deleolajide** [igniterealtime/openfire-monitoring-plugin#36]
When a message is formatted as a multi-line string with carriage return "\n" characters, it is saved correctly in the DB, but monitoring plugin removes the "\u000a" in the body of the archived message stanza created.

```
<message from="test@conference.desktop-545pc5b/Dele Olajide" id="e0678c1b-fa3d-421b-a5b3-5b39d4a8b149" to="dele@desktop-545pc5b/pade-converse-v4.2.0-Chrome73Win10.64-26059867" type="groupchat">
  <body>Your heading goes here ======================</body>
  <active xmlns="http://jabber.org/protocol/chatstates"></active>
  <origin-id xmlns="urn:xmpp:sid:0" id="e0678c1b-fa3d-421b-a5b3-5b39d4a8b149"></origin-id>
  <stanza-id xmlns="urn:xmpp:sid:0" id="e0678c1b-fa3d-421b-a5b3-5b39d4a8b149" by="test@conference.desktop-545pc5b"/>
</message>
```

This causes markdown text to be formatted incorrectly

**Comment by guusdk** [comment]
This is caused by https://issues.igniterealtime.org/browse/OF-1804

**Comment by guusdk** [comment]
Fixed in 1.8.0.
